node/src/components/consensus/protocols/highway.rs

```diff
@@ -677,7 +704,8 @@ where
             }
             TIMER_ID_PURGE_VERTICES => {
                 self.synchronizer.purge_vertices(now);
-                let next_time = now + self.synchronizer.pending_vertex_timeout();
+                self.pvv_cache.clear();
+                let next_time = Timestamp::now() + self.synchronizer.pending_vertex_timeout();
```

Comment:
I think I removed all calls to `now()` to make the unit tests deterministic. (Not sure whether this one matters, but it would still be nice to keep the wall clock out of this module.)
```suggestion
                let next_time = now + self.synchronizer.pending_vertex_timeout();
```

Reply:
Done in [`96422dc` (#1201)](https://github.com/CasperLabs/casper-node/pull/1201/commits/96422dc1c1e4ab54cd2167d68a799365ab6928ec)